web/store/chat.ts

```diff
@@ -248,6 +249,7 @@ export const chatStore = createStore<ChatState>('chat', {
       const { msgs } = msgStore.getState()
 
       const book = memoryStore.getState().books.list.find((bk) => bk._id === active.chat.memoryId)
+      const settings = presetStore.getState().presets.find((p) => p._id === active.chat.genPreset)
```

Comment:
We should probably use the `getPromptEntities` function here. It should work away.
There is some really annoying logic for fetching the correct generation settings.